changelog/0008-blob-metadata-part-1.yml

```diff
@@ -0,0 +1,12 @@
+title: Blob Metadata Migration - part 1
+key: blob-metadata-part-1
+date: 2018-09-24
+
+optional_per_env: no
+
+min_commcare_version: d7a5ee92d32f719e3e19c5841fdc7cecd2072985
+max_commcare_version:
```

Comment:
Hmm, it looks like these min/max revisions in the change log are just used to create notes in the change log to be observed by people doing upgrades. I had previously thought they were used to enforce a specific upgrade path through some automated process. With what I've done in this PR these fields are no longer referenced anywhere. Working on some ideas here.